src/quorum/majority.rs

```diff
@@ -130,6 +132,87 @@ impl Configuration {
             VoteResult::Lost
         }
     }
+
+    /// Describe returns a (multi-line) representation of the commit indexes for the
+    /// given lookuper.
+    /// Including `Index`,`Id` and the number of smaller index (represented as the bar)
+    ///
+    /// Print `?` if `Index` is not exist.
+    ///
+    /// e.g.
+    /// ```txt
+    ///             idx
+    /// x>          100 (id=1)
+    /// xx>         101 (id=2)
+    /// >            99 (id=3)
+    /// 100
+    /// ```
+    pub(crate) fn describe(&self, l: &impl AckedIndexer) -> String {
```

Comment:
Perhaps we can mark it with `#[cfg(test)]` so that we don't need another test case to test it.

Reply:
Resolved